app/components/MobileNavbar.tsx

```diff
@@ -10,6 +10,9 @@ const MobileNavbar = () => {
         <Link href="/" className="font-semibold hover:opacity-50">
           Home
         </Link>
+        <Link href="/summary" className="font-semibold hover:opacity-50">
+          Summary
```

Comment:
also, the name "Summary" seems too nondescript IMO; I wasn't exactly sure what it meant when I first saw the PR title — maybe "Course List" or "Summary Data" even? Open to suggestions